From 69bfdc576fb79c3409ba4862490d84ae887e9549 Mon Sep 17 00:00:00 2001
Message-Id: <69bfdc576fb79c3409ba4862490d84ae887e9549.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Vijay Purushothaman <vijay.a.purushothaman@intel.com>
Date: Tue, 26 Aug 2014 10:17:35 +0530
Subject: [PATCH 26/75] MUST_REBASE [VPG]: adf: Treat warning as errors for
 adf and adf/intel drivers

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

For: GMINL-3547
Change-Id: I6c43c24f436f4aa2057837a0cf46dcd9b80de118
Signed-off-by: Vijay Purushothaman <vijay.a.purushothaman@intel.com>
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/video/adf/Makefile                   |    2 +-
 drivers/video/adf/intel/Makefile             |    2 +-
 drivers/video/adf/intel/core/Makefile        |    5 +----
 drivers/video/adf/intel/core/common/Makefile |    5 +----
 4 files changed, 4 insertions(+), 10 deletions(-)

diff --git a/drivers/video/adf/Makefile b/drivers/video/adf/Makefile
index 1d11737..9949728 100644
--- a/drivers/video/adf/Makefile
+++ b/drivers/video/adf/Makefile
@@ -1,4 +1,4 @@
-ccflags-y := -Idrivers/staging/
+ccflags-y := -Idrivers/staging/ -Werror
 
 CFLAGS_adf.o := -I$(src)
 
diff --git a/drivers/video/adf/intel/Makefile b/drivers/video/adf/intel/Makefile
index d3f21a1..ad14dd9 100644
--- a/drivers/video/adf/intel/Makefile
+++ b/drivers/video/adf/intel/Makefile
@@ -1,5 +1,5 @@
 # Makefile for Intel display driver based on ADF framework
-ccflags-y := -Idrivers/staging/ -Iinclude/adf -Iinclude/uapi/adf -Idrivers/video/adf/intel/include
+ccflags-y := -Idrivers/staging/ -Iinclude/adf -Iinclude/uapi/adf -Idrivers/video/adf/intel/include -Werror
 
 obj-$(CONFIG_ADF_INTEL) += intel_adf.o \
 	intel_adf_device.o \
diff --git a/drivers/video/adf/intel/core/Makefile b/drivers/video/adf/intel/core/Makefile
index ecd468f..39929f0 100644
--- a/drivers/video/adf/intel/core/Makefile
+++ b/drivers/video/adf/intel/core/Makefile
@@ -1,8 +1,5 @@
 # Makefile for Intel display driver based on ADF framework
-
-ccflags-y := -Idrivers/staging/ -Iinclude/adf -Iinclude/uapi/adf -Idrivers/video/adf/intel/include
-
-CFLAGS_adf.o := -I$(src)
+ccflags-y := -Idrivers/staging/ -Iinclude/adf -Iinclude/uapi/adf -Idrivers/video/adf/intel/include -Werror
 
 obj-$(CONFIG_ADF_INTEL) += intel_adf_drv.o \
 	intel_plane.o \
diff --git a/drivers/video/adf/intel/core/common/Makefile b/drivers/video/adf/intel/core/common/Makefile
index c950d6e..8884604 100644
--- a/drivers/video/adf/intel/core/common/Makefile
+++ b/drivers/video/adf/intel/core/common/Makefile
@@ -1,7 +1,4 @@
 # Makefile for Intel display driver based on ADF framework
-
-ccflags-y := -Idrivers/staging/ -Iinclude/adf -Iinclude/uapi/adf -Idrivers/video/adf/intel/include
-
-CFLAGS_adf.o := -I$(src)
+ccflags-y := -Idrivers/staging/ -Iinclude/adf -Iinclude/uapi/adf -Idrivers/video/adf/intel/include -Werror
 
 obj-$(CONFIG_ADF_INTEL) += backlight_dev.o
-- 
1.7.9.5

